web/themes/custom/pineapple/templates/navigation/menu.html.twig

```diff
@@ -0,0 +1,64 @@
+{#
+/**
+ * @file
+ * Theme override to display a menu.
+ *
+ * Available variables:
+ * - menu_name: The machine name of the menu.
+ * - items: A nested list of menu items. Each menu item contains:
+ *   - attributes: HTML attributes for the menu item.
+ *   - below: The menu item child items.
+ *   - title: The menu link title.
+ *   - url: The menu link url, instance of \Drupal\Core\Url
+ *   - localized_options: Menu link localized options.
+ *   - is_expanded: TRUE if the link has visible children within the current
+ *     menu tree.
+ *   - is_collapsed: TRUE if the link has children within the current menu tree
+ *     that are not currently visible.
+ *   - in_active_trail: TRUE if the link is in the active trail.
+ */
+#}
+{% import _self as menus %}
+
+{#
+  We call a macro which calls itself to render the full tree.
+  @see https://twig.symfony.com/doc/1.x/tags/macro.html
+#}
+{{ menus.menu_links(items, attributes, 0) }}
+
+{% macro menu_links(items, attributes, menu_level) %}
+  {% import _self as menus %}
+  {% if items %}
+{# At the highest level of the menu, add div with class menu, else add a div with class sub-menu.#}
+    {% if menu_level == 0 %}
+<div{{ attributes.addClass('menu') }}>
+      <button class="hamburger">☰</button>
+    {% else %}
+  <div class="sub-menu">
+    {% endif %}
+    {# For each item in items, set the following classes.#}
+    {% for item in items %}
+      {% set classes = [
+        'menu-item',
+        item.is_expanded ? 'menu-item--expanded',
+        item.is_collapsed ? 'menu-item--collapsed',
+        item.in_active_trail ? 'menu-item--active-trail',
+      ] %}
+{# Set a variable link_attributes to equal the classes defined above. #}
+      {% set link_attributes = item.attributes.addClass(classes) %}
+{#If item has class of menu-item--expanded, add a div with a class of expanded-container.#}
+      {% if item.attributes.hasClass('menu-item--expanded') %}
+        <div class="expanded-container">
+          <a {{ link_attributes }} href="{{ item.url }}">{{ item.title }}</a>
+{#          If item has a child, add divs with the same structure as the menu on the highest level, else just add a regular link. #}
+          {% if item.below %}
+            {{ menus.menu_links(item.below, attributes, menu_level + 1) }}
+          {% endif %}
+        </div>
+      {% else %}
+        <a {{ link_attributes }} href="{{ item.url }}">{{ item.title }}</a>
+      {% endif %}
+    {% endfor %}
+  </div>
+  {% endif %}
+  {% endmacro %}
```

Comment:
Format this file correctly, comments should start where the line they pertain to start and also code should be indented correctly

Reply:
done